galley/pkg/config/meta/metadata/gen.go

```diff
@@ -18,14 +18,14 @@ package metadata
 //go:generate go-bindata --nocompress --nometadata --pkg metadata -o metadata.gen.go metadata.yaml
```

Comment:
Could you call this file something other than gen.go? We use the suffix .gen.go to indicate generated files, so we skip different linters and other things on these files. I'm worried that having a file called gen.go is just inviting confusion. generate.go would be fine instead.

Reply:
done